net.shrew.ike.qikea.json

```diff
@@ -0,0 +1,101 @@
+{
+    "app-id": "net.shrew.ike.qikea",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "command": "qikea",
+    "sdk": "org.freedesktop.Sdk",
+    "finish-args": [
+        "--filesystem=host",
```

Comment:
Does it really need host access?

Reply:
Not really, home access would be enough. Should I change it then?

Reply:
Yes please.

Reply:
Done with a5d6b3c